text/1002-required-by-default.rst

```diff
@@ -0,0 +1,178 @@
+..
+    Status: Draft
+    Type: Guideline
+    Created: 2020-06-17
+    RFC PR: `edgedb/rfcs#0001 <https://github.com/edgedb/rfcs/pull/1>`_
+
+===============================================================
+RFC 1002: REQUIRED qualifier by default in properties and links
+===============================================================
+
+This RFC describes the reasons why DDL and SDL declarations for properties
+and links assume non-optionality by default.
+
+
+Motivation
+==========
+
+The data definition language, and more importantly, the schema definition
+language, which users use to communicate with EdgeDB, need to avoid
+unnecessary verbosity.
+
+This requires some conventions and assumptions about what the user expects
+most of the time. Those assumptions shouldn't bar the user from changing
+their mind when the need arises.
+
+A default value has far reaching consequences which we cover in this
+document.
+
+
+Implementation
+==============
+
+Since the discussion always applies to properties and links, we're referring
+to them simply as *fields* below.
```

Comment:
Let's refer to the as `pointers`. That's the terminology we use for them in schema `schema::Pointer` and in our code.